src/main/java/seedu/address/model/timetable/Timetable.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.model.timetable;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import org.joda.time.LocalTime;
+/**
+ * Represents a timetable for a person.
+ */
+public class Timetable {
+    private static final String[] startTimings = new String[] {
```

Comment:
Since we’re working with hours only, can just be “08” for simplicity. Better still, can just integerise it.

Reply:
I will be pushing new corrections regarding this!

Reply:
We will be using 
```java
new LocalTime(int: hours, int: mins);
```
given the input of the data file.